recipe/meta.yaml

```
@@ -28,16 +28,19 @@ build:
 requirements:
   build:
     - {{ posix }}zip               # [win]
+    - make
```

```suggestion
    - make
    - {{ compiler('c') }}
    - {{ compiler('cxx') }}
```

And then you'll need to rerender

---

mostly just taken from the dplyr conda-forge recipe

---

@mariusvniekerk thank you for the hint and suggestion!